src/bun.js/test/jest.zig

```diff
@@ -1398,6 +1398,7 @@ pub const DescribeScope = struct {
     file_id: TestRunner.File.ID,
     current_test_id: TestRunner.Test.ID = 0,
     value: JSValue = .zero,
+    done_result: ?JSValue = null,
```

Comment:
This technically can be .zero instead of optional, but its okay

Reply:
turns out I want it to just be `bool` after all :rofl: